spec/features/custom_styles_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe "Custom style" do
+  it "is applied", :js do
+    visit admin_customers_path
+    expect(page).to_not have_css('.custom-administrate-css-hidden')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.